web/cypress/e2e/check-license-status.spec.ts

```diff
@@ -10,15 +10,16 @@ describe("check license status [feature] [all] [group1]", () => {
 
   it("searches and checks license status", () => {
     const industry = LookupIndustryById("home-contractor");
-    const legalStructureId = "limited-liability-company";
+    const legalStructureId = "general-partnership";
+    const townDisplayName = "Barnegat";
+    const businessName = "Aculyst";
 
     completeNewBusinessOnboarding({
       industry,
-      legalStructureId,
     });
```

Comment:
As a style not, now that this call is so short I'd probably one-line it instead of three lines